src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -52,22 +53,15 @@ def command_create_profile(
     _, storage_entry_point = get_entry_point_from_class(storage_cls.__module__, storage_cls.__name__)
     assert storage_entry_point is not None
 
-    if kwargs.pop('use_rabbitmq'):
-        broker_backend = 'core.rabbitmq'
-        broker_config = {
-            key: kwargs.get(key)
-            for key in (
-                'broker_protocol',
-                'broker_username',
-                'broker_password',
-                'broker_host',
-                'broker_port',
-                'broker_virtual_host',
-            )
-        }
+    broker_config = detect_rabbitmq_config()
+    broker_backend = 'core.rabbitmq' if kwargs.get('use_rabbitmq', False) else None
+
+    if broker_config is None:
```

Comment:
Isn't the information that is exposed to the user a bit confusing here in case the user specifies `--no-use-rabbitmq`? The user still sees echo's for configuring rabbitmq. I understand that `use_rabbitmq` is given by the user through `--use-rabbitmq`, so there should be no rabbitmq information in this case.
```suggestion
    if kwargs.get('use_rabbitmq', False):
        broker_config = detect_rabbitmq_config()
        broker_backend = 'core.rabbitmq'
        if broker_config is None:
            echo.echo_warning(...)
        else:
            echo.echo_success(...)
        echo.echo_report('RabbitMQ connection parameters can be reconfigured with `verdi profile configure-rabbitmq`.')         
```
I am not sure, what the behavior should be if the user does not specify the argument at all. The code above always turns it off, but it seems to me nicer if we try to set it up when it is not specified by the user. In this case if it is not available showing the warnings is okay.